File: src/main/java/com/intellij/plugin/applescript/lang/dictionary/xml/LegacyJdomWriter.java
package com.intellij.plugin.applescript.lang.dictionary.xml;

import com.intellij.openapi.util.JDOMUtil;
import org.jdom.Document;

import java.io.IOException;
import java.io.OutputStream;

public final class LegacyJdomWriter {
    private LegacyJdomWriter() {
    }

    public static void write(Document document, OutputStream output) throws IOException {
        JDOMUtil.write(document, output);
    }
}

File: src/main/kotlin/com/intellij/plugin/applescript/lang/dictionary/xml/LegacyJdomWriter.kt
package com.intellij.plugin.applescript.lang.dictionary.xml

import com.intellij.openapi.util.JDOMUtil
import org.jdom.Document
import java.io.OutputStream

/** Writes a merged JDOM [Document] back to a stream for the scripting-additions dictionary cache. */
internal object LegacyJdomWriter {
    fun write(
        document: Document,
        output: OutputStream,
    ) = JDOMUtil.write(document, output)
}
